security/pkg/nodeagent/sds/server.go

```diff
@@ -333,6 +334,12 @@ func setUpUds(udsPath string) (net.Listener, error) {
 		return nil, fmt.Errorf("failed to remove unix://%s", udsPath)
 	}
 
+	// Attempt to create the folder in case it doesn't exist
+	if err := os.MkdirAll(filepath.Dir(udsPath), 0750); err != nil {
```

Comment:
So this is only used for local run, right? I think in K8s this dir will be mounted.

Reply:
// If path is already a directory, MkdirAll does nothing
// and returns nil.